source/includes/openstack/_instances.md

```diff
@@ -147,6 +161,30 @@ Required                  | Description
 ------------------------- | -------------------------------------------
 `floatingIpId`<br/>*UUID* | The ID of a previously-acquired floating IP
 
+#### Change security groups
+```shell
+curl -X POST \
+    -H "MC-Api-Key: your_api_key" \
+    -H "Content-Type: application/json" \
+    -d "request_body" \
+    "https://api.your.cloudmc/v1/services/compute-os/devel/instances/30fca349-68b0-48c2-9ada-1f60f57fa44e?operation=changeSecurityGroups"
+# Request body should look like
+```
+```json
+{
+  "newSecurityGroups": [ "securityGroupNameA", "securityGroupNameB"]
```

Comment:
Remember to change this to `securityGroupNames` before merging (if you implement that change in the code).